AnyQt/QtCore.py

```diff
@@ -298,6 +298,21 @@
     del _QtCore, _QtGui, _major, _minor, _micro
     # Known to be in PyQt4 but missing in PySide:
     #     Q_ARG, Q_CLASSINFO, Q_ENUMS, Q_FLAGS, Q_RETURN_ARG, ...
+elif _api.USED_API == _api.QT_API_PYSIDE2:
+    from PySide2 import QtCore as _QtCore
+    globals().update(
+        {name: getattr(_QtCore, name)
+         for name in __Qt4_QtCore if hasattr(_QtCore, name)}
```

Comment:
This should probably import full namespace from PySide2.QtCore. Currently the `QAbstractProxyModel`, `QItemSelection`, ... are not re-exported (at the least it should export the combined `__Qt4_QtCore + __Qt4_QtGui` names).  Note also that `QStringListModel` still seems to be in PySide2.QtGui.

Reply:
I reported the QStringListModel issue upstream, I'm testing a pathc so we dont have to fix it here as no stable version is out yet:
https://bugreports.qt.io/projects/PYSIDE/issues/PYSIDE-614